components/brave_vpn/browser/connection/ikev2/brave_vpn_ras_connection_api_base.h

```diff
@@ -71,6 +71,11 @@ class BraveVPNOSConnectionAPIBase
   virtual void RemoveVPNConnectionImpl(const std::string& name) = 0;
   virtual void CheckConnectionImpl(const std::string& name) = 0;
 
+  BraveVpnAPIRequest* GetAPIRequest();
+  std::string GetCurrentEnvironment() const;
+  void UpdateAndNotifyConnectionStateChange(mojom::ConnectionState state);
```

Comment:
why these are moved from private to here?

Reply:
good catch, moved it back